src/audio/drc/drc_math_lib/asin/matlab_fixpt/init_struc_fixpt.m

```diff
@@ -0,0 +1,13 @@
+function x = init_struc_fixpt ()
```

Comment:
there should not be any matlab code under src/ unless there is a good reason, we should move this under toosl/tune

Reply:
This was done to have a testbench code  , I am ok to your suggestion, I will move the  all matlab floating and fixed scripts under tools/tune folder